login-tool/tool/src/java/org/sakaiproject/login/tool/SkinnableLogin.java

```diff
@@ -422,13 +422,19 @@ public LoginRenderContext startPageContext(String skin, HttpServletRequest reque
 		String localCSS = serverConfigurationService.getString("local.css", null);
 		String uiService = serverConfigurationService.getString("ui.service", "Sakai");
 		String passwordResetUrl = getPasswordResetUrl();
+		String ssoWording = serverConfigurationService.getString("xlogin.sso", null);
+		String ssoLinkText = serverConfigurationService.getString("login.text");
+		String loginWebauthUrl = serverConfigurationService.getString("login.webauth.url");
```

Comment:
webauth is an Oxford term, we should be keeping things like this out of the code so we can feed it back.